keyboards/ergotravel/keymaps/xyverz/keymap.c

```diff
@@ -0,0 +1,134 @@
+    KC_ESC,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_PGDN, KC_END,  KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_LGUI, KC_ENT,  KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
+    KC_LCTL, KC_LALT, KC_LEFT, KC_RGHT,          LOWER,   KC_BSPC, KC_SPC,  RAISE,            KC_UP,   KC_DOWN, KC_RGUI, KC_RCTL
+  ),
+
+  [_COLEMAK] = LAYOUT ( /* colemak */
+    KC_TAB,  KC_Q,    KC_W,    KC_F,    KC_P,    KC_G,    KC_PGUP, KC_HOME, KC_J,    KC_L,    KC_U,    KC_Y,    KC_SCLN, KC_DEL,
+    KC_ESC,  KC_A,    KC_R,    KC_S,    KC_T,    KC_D,    KC_PGDN, KC_END,  KC_H,    KC_N,    KC_E,    KC_I,    KC_O,    KC_QUOT,
+    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_LGUI, KC_ENT,  KC_K,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
+    KC_LCTL, KC_LALT, KC_LEFT, KC_RGHT,          LOWER,   KC_BSPC, KC_SPC,  RAISE,            KC_UP,   KC_DOWN, KC_RGUI, KC_RCTL
+  ),
+
+  [_LOWER] = LAYOUT ( /* lower */
+    KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, _______, _______, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, _______,
+    KC_CAPS, _______, KC_MUTE, KC_VOLD, KC_VOLU, _______, _______, _______, _______, _______, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE,
+    _______, _______, KC_MPRV, KC_MPLY, KC_MNXT, _______, _______, _______, _______, _______, KC_PSCR, KC_SLCK, KC_PAUS, _______,
+    _______, _______, KC_HOME, KC_END,           _______, KC_DEL,  KC_INS,  _______,          KC_PGUP, KC_PGDN, _______, _______
+  ),
+
+  [_RAISE] = LAYOUT ( /* raise */
+    KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    _______, _______, KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    _______,
+    KC_CAPS, _______, KC_MUTE, KC_VOLD, KC_VOLU, _______, _______, _______, _______, _______, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+    _______, _______, KC_MPRV, KC_MPLY, KC_MNXT, _______, _______, _______, _______, _______, KC_PSCR, KC_SLCK, KC_PAUS, _______,
+    _______, _______, KC_HOME, KC_END,           _______, KC_DEL,  KC_INS,  _______,          KC_PGUP, KC_PGDN, _______, _______
+  ),
+
+  [_ADJUST] = LAYOUT (
+    KC_F11,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   _______, _______, KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F12,
+    _______, RESET,   _______, _______, _______, _______, _______, _______, _______, QWERTY,  COLEMAK, DVORAK,  _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______,          _______, _______, _______, _______,          _______, _______, _______, _______
+  ),
+
+};
+
+void matrix_init_user(void) {
+#ifdef BOOTLOADER_CATERINA
+   // This will disable the red LEDs on the ProMicros
+   DDRD &= ~(1<<5);
+   PORTD &= ~(1<<5);
+   DDRB &= ~(1<<0);
+   PORTB &= ~(1<<0);
+#endif
+};
+
+const uint16_t PROGMEM fn_actions[] = {
```

Comment:
Lines 84-85 should be deleted. This method is deprecated, and your keymap isn't using it anyway.

Reply:
Thanks!